packages/synapse-react-client/src/components/Markdown/MarkdownGithub.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react'
+import { SkeletonParagraph } from '../Skeleton'
+import MarkdownSynapse from './MarkdownSynapse'
+import { Box } from '@mui/material'
+import { useLatestTag } from '../../synapse-queries/github/useGithub'
+
+export type MarkdownGithubProps = {
```

Comment:
SWC to use this component to show all documents from the Governance TrustCenter repo.